likelihood/cl_like/cl_like.py

```diff
@@ -241,10 +243,14 @@ def _get_tracers(self, cosmo, **pars):
                 bz = self._get_bz(cosmo, name, **pars)
                 t = ccl.NumberCountsTracer(cosmo, dndz=nz, bias=bz, has_rsd=False)
                 if self.bz_model == 'EulerianPT':
+                    z = self.bin_properties[name]['z_fid']
+                    zmean = self.bin_properties[name]['zmean_fid']
                     b1 = pars[self.input_params_prefix + '_' + name + '_b1']
+                    bp = pars[self.input_params_prefix + '_' + name + '_bp']
```

Comment:
Just in case we want to study the evolution of the other bias parameters, let's name this `b1p` instead of just `bp` (that way we can include `b2p` etc. in the future if we need to)